README.md

```
@@ -154,7 +154,112 @@ For Linux-specific builds (Flatpak, AppImage), see [guides/linux.md](guides/linu
+# Your Convex deployment (from dashboard, e.g., "dev:your-project-name")
+CONVEX_DEPLOYMENT=dev:your-project-name
+
+# For Electron app development, use app:// protocol
+SITE_URL=app://-
+
+# Generate a secure secret for production (optional for dev)
+BETTER_AUTH_SECRET=your-secure-random-string-here
+```
+
+**To generate a secure secret:**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)